index.html

```diff
@@ -0,0 +1,228 @@
+                    <div class="prog-title-container">
+                        <h3 class="prog-title">
+                            Forum
+                        </h3>
+                    </div>
+                    <div class="prog-description">
+                        <p>
+                            Have the time to share yur thoughts and 
+                            opnions with experts for each topic.
+                        </p>
+                    </div>
+                </div>
+                <div class="prog-item prog-bkgd ">
+                    <div class="prog-img">
+                        <img class="prog-icon" src="./images/workshop-icon.png" alt="The icon of a lecturer icon">
+                    </div>
+                    <div class="prog-title-container">
+                        <h3 class="prog-title">
+                            Workshop
+                        </h3>
+                    </div>
+                    <div class="prog-description">
+                        <p>
+                            Try creating your won work suing open source 
+                            license rather than just watching
+                        </p>
+                    </div>
+                </div>
+                <div class="prog-item prog-bkgd">
+                    <div class="prog-img">
+                        <img class="prog-icon" src="./images/cc-ignite-icon.png" alt="The icon of a lecturer icon">
+                    </div>
+                    <div class="prog-title-container">
+                        <h3 class="prog-title">
+                            CC Ignite
+                        </h3>
+                    </div>
+                    <div class="prog-description">
+                        <p>
+                            Get opportunities to network with CC affiliates around
+                            the world, also after the summit.
+                        </p>
+                    </div>
+                </div>
+            </div>
+            <!--See whole Program link -->
+            <div class="whole-program-link">
+                  <p class="whole-program-link-text">
+                    <a href="#">
+                        SEE THE WHOLE PROGRAM
+                    </a>
+                  </p>  
+                
+            </div>
+            <div class="prog-registration-button">
+                <p class="prog-reg-text">
+                    Join CC Global Sumbit 2023
+                </p>
+            </div>
+        </section> 
+        <!--Speakers Section-->
+        <section class="speakers-section prog-bkgd blur">
+            <div class="section-heading ">
+                <h2 class="section-head dark-color">
+                    Featured Speakers
+                </h2>
+                <hr class="section-heading-hr speaker-head-hr">
+            </div>
+            <!--dynamic content here -->
+            <div id="the-spkrs" class="featured-speakers">
+            </div>
```

Comment:
[REQUIRED] In your mobile screen size some of your featured speakers data is hidden beneath the partners section.
![image](https://user-images.githubusercontent.com/62938735/234921260-b17f84ab-1a3a-4799-8c7c-4ba203520dfe.png)

Reply:
Thanks. Done

Reply:
- [ ] [OPTIONAL] Good job so far! It would be nice if you implement a `More` button end of this section 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/235318413-0ab70fd4-1865-44c3-b36e-14d999a441df.png)

Reply:
Sorry, I miss committing that. I will surely add it to the project. Thank you.